views/pagination.templ

```diff
@@ -6,49 +6,49 @@ import "github.com/ugent-library/biblio-backoffice/ctx"
 import "github.com/ugent-library/biblio-backoffice/models"
 
 templ Pagination(c *ctx.Ctx, baseURL *url.URL, searchArgs *models.SearchArgs, searchHits *models.SearchHits) {
-    <ul class="pagination">
-    if searchHits.HasPreviousPage() {
-        <li class="page-item">
-            <a class="page-link" href={templ.URL(urlWithQuery(baseURL, searchArgs.Clone().WithPage(searchHits.PreviousPage())).String())} aria-label="Previous">
-                <i class="if if-chevron-left" aria-hidden="true"></i>
-            </a>
-        </li>
-    } else {
-        <li class="page-item disabled">
-            <a class="page-link" href="#" aria-label="Previous">
-                <i class="if if-chevron-left" aria-hidden="true"></i>
-            </a>
-        </li>
-    }
-    for _, page := range searchHits.PagesWithEllipsis() {
-        if page > 0 {
-            <li class={ "page-item", templ.KV("active", searchHits.Page() == page) }>
-                <a class="page-link" href={templ.URL(urlWithQuery(baseURL, searchArgs.Clone().WithPage(page)).String())} aria-label={fmt.Sprintf("Page %d", page)}>
-                    {fmt.Sprintf("%d", page)}
-                </a>
-            </li>
-        } else {
-            <li class="page-item disabled">
-                <a class="page-link" href="#">
-                    &hellip;
-                </a>
-            </li>
-        }
-    }
-    if searchHits.HasNextPage() {
-        <li class="page-item">
-            <a class="page-link" href={templ.URL(urlWithQuery(baseURL, searchArgs.Clone().WithPage(searchHits.NextPage())).String())} aria-label="Next">
-                <i class="if if-chevron-right" aria-hidden="true"></i>
-            </a>
-        </li>
-    } else {
-        <li class="page-item disabled">
-            <a class="page-link" href="#" aria-label="Next">
-                <i class="if if-chevron-right" aria-hidden="true"></i>
-            </a>
-        </li>
-    }
-    </ul>    
+	<ul class="pagination">
+		if searchHits.HasPreviousPage() {
+			<li class="page-item">
+				<a class="page-link" href={ NewURLBuilder(baseURL.String()).WithQuery(searchArgs.Clone().WithPage(searchHits.PreviousPage())).SafeURL() } aria-label="Previous">
```

Comment:
Already implemented for the pagination template but this is only used for candidate records which I don't know how to test. Can you test that, @nicolasfranck?